src/components/Breadcrumb/index.tsx

```diff
@@ -21,7 +21,7 @@ export const BreadCrumb: React.FC<Props> = ({
     <StyledBreadcrumb>
       <Breadcrumb.Item>
         <StyledP1>
-          <StyledRouterLink id="bc-my-patches" to={paths.myPatches}>
+          <StyledRouterLink id="bc-my-patches" to={routes.myPatches}>
```

Comment:
can we change this route to `userPatches` ?

Reply:
there already is a userPatches route